game_patch/hud/multi_scoreboard.cpp

```diff
@@ -53,14 +53,17 @@ int draw_scoreboard_header(int x, int y, int w, rf::NetGameType game_type, bool
 
     // Draw Game Type name
     if (!dry_run) {
-        const char* game_type_name;
-        if (game_type == rf::NG_TYPE_DM)
-            game_type_name = rf::strings::deathmatch;
-        else if (game_type == rf::NG_TYPE_CTF)
-            game_type_name = rf::strings::capture_the_flag;
-        else
-            game_type_name = rf::strings::team_deathmatch;
-        rf::gr::string_aligned(rf::gr::ALIGN_CENTER, x_center, cur_y, game_type_name);
+        int num_players = rf::multi_num_players();
+        std::string player_count_str =
+            " | " + std::to_string(num_players) + (num_players > 1 ? " PLAYERS" : " PLAYER");
+
+        std::string game_type_name = (game_type == rf::NG_TYPE_DM)    ? rf::strings::deathmatch
+                                     : (game_type == rf::NG_TYPE_CTF) ? rf::strings::capture_the_flag
+                                                                      : rf::strings::team_deathmatch;
+
+        game_type_name += player_count_str;
+
+        rf::gr::string_aligned(rf::gr::ALIGN_CENTER, x_center, cur_y, game_type_name.c_str());
```

Comment:
#306 paves the way for introducing an identifier for browser clients. Let's see what rafalh says.

Reply:
I totally agree with Goober that without a clear indication what we count it will be confusing to skip browsers.
Still, even if we add some image besides browser clients it will still be a bit confusing if people see a different count in server list screen and after joining the server. Maybe we should display two counters: number of all players, and number of browsers or add some small hint that the counter does not include browsers?
Anyway why we want to skip browsers? If we care about players actively playing the game AFK players could also be treated differently? 
I think we should prevent scope-creep in this PR and leave the browsers issue to another one

Reply:
About the proposed code, it looks ok, but I would prefer a standalone function over `std::invoke` with a lambda here

Reply:
An idea could be to display a number of player slots occupied and separately, a number of currently spawned players?

I don't know how to appropriately communicate that on the UI though - there's not a lot of room to work with.

Agreed though that scope creep should be prevented here. Anything beyond displaying a number of occupied player slots should be out of scope for this simple change IMO.

Reply:
Perhaps it could be `num_spawned_players/num_players PLAYING` e.g. `5/8 PLAYING`.

Reply:
sounds good, although I'm not sure if counting alive people "as is" is the best approach because people spawn and die all the time...
to be honest i'm not sure if clobbering panel with multiple statistics is worth it considering it can reduce readability. For me it's okay to just display total player count which in most cases is enough, but if you add "playing" I will accept that too